gravitee-apim-portal-webui-next/src/entities/categories/categories.fixture.ts

```diff
@@ -48,3 +48,26 @@ export function fakeCategoriesResponse(modifier?: Partial<Categories> | ((baseCa
     ...modifier,
   };
 }
+
+export function fakeCategory(modifier?: Partial<Category> | ((baseCategory: Category) => Category)): Category {
+  const base: Category = {
+    id: 'category-1',
+    name: 'Category 1',
+    description: 'My first category',
+    order: 0,
+    total_apis: 4,
+    _links: {
+      picture: 'test-picture',
+      background: 'test-background',
+    },
+  };
+
+  if (isFunction(modifier)) {
+    return modifier(base);
+  }
+
+  return {
+    ...base,
+    ...modifier,
```

Comment:
If you like you can use this form
return isFunction(modifier) ? modifier(base) : { ...base, ...modifier };

Reply:
Done